docs/UserGuide.md

```diff
@@ -142,16 +142,16 @@ Edits an existing person in the contact list of FINDvisor.
 
 Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
+* Edits the person at the specified `INDEX`. The index refers to the index number shown in the current displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
```

Comment:
I think the previous description is sufficient?

Reply:
The new description seems more specific but I think we should add that it is for the "specified field" instead of just field.

Reply:
Okay! I have changed it! Thanks